src/test/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecuteConfigTest.java

```diff
@@ -47,11 +48,29 @@ public void testBigQueryExecuteValidSQL() throws Exception {
     Assert.assertEquals(0, failureCollector.getValidationFailures().size());
   }
 
+  @Test
+  public void testBigQueryExecuteSQLWithNonExistentResource() throws Exception {
+    String errorMessage = "Not found";
+    BigQueryExecute.Config config = new BigQueryExecute.Config("standard",
+            "select * from dataset.table where id=1",
+            "batch");
+
+    FieldSetter.setField(config, GCPConfig.class.getDeclaredField("project"), "test_project");
```

Comment:
Please don't set fields like this, it is brittle (like changing the field name will break the test)!
Generally, we use a builder to create the config and set it normally.

Reply:
Addressed this in the new rev